primer.md

```diff
@@ -174,6 +175,15 @@ attributes may be defined that facilitate meeting persistence requirements but
 it is expected that these will continuously evolve along with industry best
 practices and advancements.
 
+The CloudEvents spec currently does not mandate or advocates any specific
+mechanism or principles in the matter of Authorization, Data Integrity and
+Confidentiality, as the core intention behind this spec is not to define
```

Comment:
I totally agree with the sentiment here.  I appreciate you putting in the effort to craft its statement - thank you.  Regarding this specific wording, I'm uncomfortable with the statement that the core intention of the spec excludes authorization/integrity/confidentiality.

I would suggest that those are core concerns but also that the diversity and necessary ongoing evolution of valid user preferences in that decision space produces the circumstance that while we allow for and perhaps help guide users to solve those requirements we are also not yet specifying the mechanisms they should or must use for solving them.  I would guess that current consensus (as you capture) expects that more standardized solutions for these concerns will enter the spec as unofficial and then official extensions that may never reach the core spec unless the whole industry can permanently agree on a single set mechanisms for these needs.  I rate the probability of that as approaching zero though so that leaves us with where we are.

Reply:
How about if we state that the **Currently, the core intention.....and is subject to change in the future**

Reply:
The core intention of the spec, to me, includes supporting users in satisfying their authorization, integrity, and confidentiality requirements.  This stands in contrast with its current state which neither offers nor mandates any mechanisms for satisfying those requirements.  FWIW, I am a little concerned I am being overly pedantic and I would be happy enough to back off the point.